java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/nodes/LoadDataPlan.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2021 4Paradigm
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com._4paradigm.openmldb.batch.nodes
+
+import com._4paradigm.hybridse.sdk.HybridSeException
+import com._4paradigm.hybridse.vm.PhysicalLoadDataNode
+import com._4paradigm.openmldb.batch.{PlanContext, SparkInstance}
+import org.slf4j.LoggerFactory
+
+import scala.collection.mutable
+
+object LoadDataPlan {
+  private val logger = LoggerFactory.getLogger(this.getClass)
+
+  def parseOptions(node: PhysicalLoadDataNode): (String, Map[String, String]) = {
+    var format = "csv"
+    var option = node.GetOption("format")
+    if (option != null) {
+      val f = option.GetStr()
+      if (f.equalsIgnoreCase("parquet")) {
```

Comment:
We can use `match` syntax and switch-case in Scala.

Reply:
ok

Reply:
There's no need to match-case, options["format"] is a string, just need to get lowercase and assign it to return value.